pages/search/search.jsx

```diff
@@ -4,11 +4,15 @@ import qs from "qs";
 import DebounceInput from 'react-debounce-input';
 import ReactGA from 'react-ga';
 import SearchTabGroup from '../../components/search-tab-group/search-tab-group.jsx';
+import Service from '../../js/service';
 
 class Search extends React.Component {
```

Comment:
I would remove dropdown related code from this file and make it a component (ie. the existing`components/help-dropdown/help-dropdown.jsx` though this file is currently not being used anywhere). State like `helpOptions` is irreverent to the Search page component (Search page only needs to know the selected help type) and value updates to `this.state.helpOptions` will cause the Search page to re-render which isn't something we want.

Reply:
Selection in the HelpDropdown component will have to trigger Search page's state update. You can use this [article](https://ourcodeworld.com/articles/read/409/how-to-update-parent-state-from-child-component-in-react) as a reference to get you started.

Reply:
ah ok 🙃 thanks for the article!